tests/profile.test.js

```diff
@@ -0,0 +1,299 @@
+import '@babel/polyfill';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import status from '../src/helpers/constants/status.codes';
+import server from '../src/index';
+
+chai.use(chaiHttp);
+chai.should();
+
+
+const data = {
+  country: 'congo',
+  firstName: 'alain',
+  lastName: 'burindi',
+  gender: 'man',
+  phoneNumber: '+243974690917',
+  address: 'gachinjiro/kigali',
+  profileImage: 'noimage.jpg',
+  bio: 'I work at statefarm',
+  facebook: 'https://facebook.com/alainburindi',
+  twitter: 'https://twitter.com/alainburindi',
+};
+
+const errors = {
+  country: 'The country is required should only contain alphanumeric characters',
+  firstName: 'The firstName should only contain alphanumeric characters',
+  lastName: 'The lastName should only contain alphanumeric characters',
+  address: 'The address is required',
+  gender: 'The gender should only contain alphanumeric characters',
+  phoneNumber: 'The phone number is required',
+  bio: 'The bio is required',
+  user: 'User doesn\'t exist',
+};
+
+const userData = {
+  username: 'myname',
+  email: 'myname@mail.com',
+  password: 'password111',
+};
+
+const {
+  country, firstName, lastName,
+  address, gender, profileImage, phoneNumber, bio
+} = data;
+
+let authToken = '';
+
+describe('Profile', () => {
+  it('should signup to get the token', (done) => {
```

Comment:
@alainburindi can we have a signup/sign-in helper method for testing purposes which everyone who's working on an authenticated route can then use?